CONTRIBUTING.txt

```diff
@@ -449,7 +449,7 @@ followed to warn users.
 
     * adding a new function, or
     * adding a new keyword argument to the *end* of a function signature, or
-    * fixing what was buggy behaviour
+    * fixing what was buggy behavior
```

Comment:
American spell checker, clearly ;) All good!

Reply:
Haha, yes. Just so it's the same spelling as the behavior a few lines above :)